mobile-app/app/screens/AppNavigator/screens/Balances/components/Announcements.tsx

```diff
@@ -15,33 +18,79 @@ export function Announcements (): JSX.Element {
   const {
     language
   } = useLanguageContext()
-  if (isSuccess && announcements !== undefined && announcements?.length > 0) {
-    const announcement = findAnnouncementForVersion(nativeApplicationVersion ?? '0.0.0', announcements, language)
-    if (announcement !== undefined) {
-      return (
-        <ThemedView
-          testID='announcements_banner'
-          style={tailwind('px-4 py-3 flex-row items-center')} light={tailwind('bg-warning-50')}
-          dark={tailwind('bg-darkwarning-50')}
-        >
-          <ThemedIcon style={tailwind('mr-2')} iconType='MaterialIcons' name='campaign' size={22} />
-          <ThemedText style={tailwind('text-xs flex-auto')} testID='announcements_text'>
-            {announcement}
-          </ThemedText>
-        </ThemedView>
-      )
-    }
+
+  const announcement = findAnnouncementForVersion(nativeApplicationVersion ?? '0.0.0', language, announcements)
+
+  if (!isSuccess || announcements === undefined || announcements.length === 0 || announcement === undefined) {
+    return <></>
+  }
+
+  if (announcement.url !== undefined && announcement.url.length !== 0) {
+    return (
+      <ThemedTouchableOpacity
+        testID='announcements_banner'
+        style={tailwind('px-4 py-3 flex-row items-center')} light={tailwind('bg-warning-50')}
+        dark={tailwind('bg-darkwarning-50')}
+        onPress={async () => await openURL(announcement.url)}
+      >
+        <ThemedIcon style={tailwind('mr-2')} iconType='MaterialIcons' name='campaign' size={22} />
+        <ThemedText style={tailwind('text-xs flex-auto')} testID='announcements_text'>
+          {`${announcement.content} `}
+          <View>
+            <ThemedIcon
```

Comment:
Can we make this fixed on the right side? Not at the end of the content but as a fixed button on the right (if link is existing)

Reply:
moved to right and increased to size `24`, but still pending final design from jonathan